doc/_templates/layout.html

```diff
@@ -9,7 +9,7 @@
 
   (function() {
     var ga = document.createElement('script'); ga.type = 'text/javascript'; ga.async = true;
-    ga.src = ('https:' == document.location.protocol ? 'https://ssl' : 'http://www') + '.google-analytics.com/ga.js';
+    ga.src = 'https://ssl.google-analytics.com/ga.js';
```

Comment:
@slefrancois What do you think of that? I think it is good, but I'm not 100% sure.